index.html

```diff
@@ -0,0 +1,93 @@
+				<nav id="nav-bar">
+					<ul id="menu-list">
+					<!-- The comment tags below is use to remove inline-block space -->
+						<li class="menu-button"><button><i class="fab fa-apple"></i></button></li><!-- 
+						 --><li class="menu-button"><button>Store</button></li><!-- 
+						  --><li class="menu-button"><button>Mac</button></li><!-- 
+						  --><li class="menu-button"><button>iPod</button></li><!-- 
+						  --><li class="menu-button"><button>iPhone</button></li><!-- 
+						  --><li class="menu-button"><button>iPad</button></li><!-- 
+						  --><li class="menu-button"><button>iTunes</button></li><!-- 
+						  --><li class="menu-button"><button>Support</button></li><!-- 
+						  --><li class="menu-button search-input"><input type="text" name="search"><i class="fas fa-search"></i></li>
+					</ul>
+				</nav>
+				<div id="main-links">
+					<div id="first-link" class="link">
+						<a href="#"> <img src="images/first.png" alt="main-link-image"></a>
+					</div>
+					<div class="link">
+						<a href="#"><img src="images/second.jpg" alt="main-link-image"></a>
+					</div>
+					<div id="last-link" class="link">
+						<img src="images/third-1.png" alt="main-link-image">
+					</div>
+				</div>
+			</div>
+			<div id="page-banner">
+				<div id="banner-image">
+					<!-- <img src="images/your_verse_hero.jpg" alt="page-background-image"> -->
+				</div>
+				<div id="image-1" class="promo-image">
+					<img src="images/promo_iphone5s.jpg" alt="iPhone5c-image">
+				</div>
+				<div id="image-2" class="promo-image">
+					<img src="images/promo_iphone5c.jpg" alt="iPhone5s-image">
+				</div>
+				<div id="image-3" class="promo-image">
+					<img src="images/promo_30_years.jpg" alt="30-year-annivesary-image">
+				</div>
+				<div id="image-4" class="promo-image">
+					<img src="images/promo_supplier_responsibility.jpg" alt="-image-of-a-worker">
+				</div>
+			</div>
+		</div>
+	</main>
+	<footer>
+		<div id="footer-wrapper">
+			<div id="footer-links">
+			<div class="online-store-links">
+	          Shop the<a href="#"> Apple Online Store </a> (1-800-MY-APPLE), visit an <a href="#"> Apple Retail Store </a>, or find a <a href="#">reseller</a>.
+	        </div>
+	        <div class="site-map-links">
+	          <ul>
+	          	 <li><a href="#">Site Map</a> | </li>
+		         <li><a href="#">Hot News</a> | </li>
+		         <li><a href="#">RSS Feeds</a> | </li>
+		         <li><a href="#">Media Info</a> | </li>
+		         <li><a href="#">Environment</a> | </li>
+		         <li><a href="#">Job Opportunities</a> | </li>
+		         <li><a href="#">Contact Us</a> | </li>
+		         <li><a href="#"><img src="icons/usa.png" alt="usa-flag"></a></li>
+	          </ul>
+	        </div>
+		</div>
+        <div class="copyright-link">
+          Copyright © 2014 Apple Inc. All rights reserved.
+          <a href="#">Terms of Use</a> |
+          <a href="#">Privacy Policy</a>
+        </div>
+		</div>
+	</footer>
+
+</body>
+</html>
+
```

Comment:
`SUGGESTION`

Please, fix the indentation. :blush:
Check [this comment]() on formattting code with editors.